grunt/config/replace.js

```diff
@@ -84,25 +84,21 @@ module.exports = function (grunt, options) {
     dist: {
       options: {
         silent: true,
-        patterns: [
-          {
-            json: function (done) {
-              done(generatePatterns());
-            }
+        patterns: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?

Reply:
@chetan-hajare this is still outstanding